themes/vimeonova/template.html

```diff
@@ -23,6 +23,7 @@
   }
 
   media-controller {
+    display: block;
```

Comment:
this fixed a weird bug where Chrome would just crash completely and show an error page with error code 5 when starting playback on the Vimeonova theme. Only on the home page.